admin-dev/themes/new-theme/scss/pages/_modules.scss

```diff
@@ -611,30 +683,51 @@ form#importDropzone {
     text-align: center;
     vertical-align: middle;
     img {
-      max-width: 57px;
+      max-width: 40px;
     }
   }
 
-  .module-column-infos {
+  .module-column-infos,
+  .module-column-description {
     display: table-cell;
     height: 50px;
     vertical-align: middle;
+    font-size:.95em;
     .module-name {
-      font-size: 1.2em;
+      color: #25b9d7;
```

Comment:
Consider using variables for colors. Some may have already been defined in this file: https://github.com/PrestaShop/PrestaShop/blob/develop/admin-dev/themes/new-theme/scss/config/_settings.scss

Reply:
:ok_hand: